src/main/java/org/datadog/jmxfetch/App.java

```diff
@@ -495,40 +500,68 @@ public boolean addJsonConfig(String name, String json) {
 
     private ConcurrentHashMap<String, YamlParser> getConfigs(AppConfig config) {
         ConcurrentHashMap<String, YamlParser> configs = new ConcurrentHashMap<String, YamlParser>();
-        YamlParser fileConfig;
 
+        loadFileConfigs(config, configs);
+        loadResourceConfigs(config, configs);
+
+        LOGGER.info("Found " + configs.size() + " config files");
+        return configs;
+    }
+
+    private void loadFileConfigs(AppConfig config, ConcurrentHashMap<String, YamlParser> configs) {
```

Comment:
Well, 'load files' was a preexisting functionality - I've just refactored some code into this helper function.
The new thing is below - loading files via resources. And this is kind of useful because this allows APM agent to package resource with itself and pass it to jmxfetch without too much hassle.